src/app/app.module.ts

```diff
@@ -194,6 +197,7 @@ export function createTranslateLoader(http: HttpClient) {
     TosComponent,
     CheckboxFieldComponent,
     NgOptimizedImage,
+    ToastrModule
```

Comment:
Glaube es macht in diesem Fall keinen Unterschied, steht aber so in der Doku:
```suggestion
    ToastrModule.forRoot(),
```